Migrations/MyPlaceDbContext.cs

```diff
@@ -0,0 +1,27 @@
+using rexfinder_api.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace rexfinder_api.Migrations;
+
+public class MyPlaceDbContext : DbContext
+{
+    public DbSet<MyPlace> MyPlaces { get; set; }
+    public MyPlaceDbContext(DbContextOptions<MyPlaceDbContext> options)
+        : base(options)
+    {
+    }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder) {
+        base.OnModelCreating(modelBuilder);
+
+        modelBuilder.Entity<MyPlace>(entity =>
+        {
+            entity.HasKey(e => e.MyPlaceId);
+            entity.Property(e => e.Visited);
+            entity.Property(e => e.UserId);
+            entity.Property(e => e.CreatedOn).IsRequired();
+            entity.Property(e => e.GooglePlaceId).IsRequired();
```

Comment:
If we are not going to use the GooglePlaces API, I should change this property name to something more generic like "ApiPlaceId" or something.

Reply:
Stick with the googleplaceID at this point please. :)